### PR TITLE
428 - Test coverage: GetAvailableShippingMethodsTest

### PR DESCRIPTION
**Pull request by atwixfirster**
**Issue:** https://github.com/magento/graphql-ce/issues/428
1. Split \Magento\GraphQl\Quote\GetAvailableShippingMethodsTest into 2 scenarions
- \Magento\GraphQl\Quote\Customer\GetAvailableShippingMethodsTest
- \Magento\GraphQl\Quote\Guest\GetAvailableShippingMethodsTest

### CODE REVIEW

File: dev/tests/integration/testsuite/Magento/Checkout/_files/disable_all_active_shipping_methods.php [hunk 1]
```diff
@@ -0,0 +1,39 @@
+<?php
```
Comment (naydav) [inline dev/tests/integration/testsuite/Magento/Checkout/_files/disable_all_active_shipping_methods.php]: Please, reuse https://github.com/magento/graphql-ce/blob/2.3-develop/dev/tests/integration/testsuite/Magento/OfflineShipping/_files/enable_offline_shipping_methods.php
fixture

**Review by naydav — CHANGES_REQUESTED** [review]

File: dev/tests/integration/testsuite/Magento/Checkout/_files/disable_all_active_shipping_methods.php (hunk 1 above)
Comment (atwixfirster) [reply dev/tests/integration/testsuite/Magento/Checkout/_files/disable_all_active_shipping_methods.php]: **done**, sir

@naydav , could you please verify? 

Thank you!

File: dev/tests/api-functional/testsuite/Magento/GraphQl/Quote/Customer/GetAvailableShippingMethodsTest.php [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+        string $maskedQuoteId
+    ): string
+    {
+        return <<<QUERY
+query {
+  cart (cart_id: "{$maskedQuoteId}") {
+    shipping_addresses {
+        available_shipping_methods {
+          amount
+          base_amount
```
Comment (naydav) [inline dev/tests/api-functional/testsuite/Magento/GraphQl/Quote/Customer/GetAvailableShippingMethodsTest.php:162]: Please, check scheme
Looks like we don't have these fields in scheme declaration

File: dev/tests/integration/testsuite/Magento/Sales/_files/guest_quote_with_addresses.php [hunk 3]
```diff
@@ -14,7 +14,7 @@
 $product = $objectManager->create(\Magento\Catalog\Model\Product::class);
 $product->setTypeId('simple')
     ->setAttributeSetId($product->getDefaultAttributeSetId())
-    ->setName('Simple Product')
+    ->setName('Simple Product For Guest Quote')
```
Comment (naydav) [inline dev/tests/integration/testsuite/Magento/Sales/_files/guest_quote_with_addresses.php]: We could not change current fixtures
It's BIC and could lead to some failures in already created tests

**Review by naydav — CHANGES_REQUESTED** [review]

File: dev/tests/api-functional/testsuite/Magento/GraphQl/Quote/Customer/GetAvailableShippingMethodsTest.php (hunk 2 above)
Comment (atwixfirster) [reply dev/tests/api-functional/testsuite/Magento/GraphQl/Quote/Customer/GetAvailableShippingMethodsTest.php:162]: > Looks like we don't have these fields in scheme declaration

@naydav , could you please check `app/code/Magento/QuoteGraphQl/etc/schema.graphqls, lines` 197-206.

File: dev/tests/integration/testsuite/Magento/Sales/_files/guest_quote_with_addresses.php (hunk 3 above)
Comment (atwixfirster) [reply dev/tests/integration/testsuite/Magento/Sales/_files/guest_quote_with_addresses.php]: > It's BIC and could lead to some failures in already created tests

Travis has passed this change. But If you recommend to not change current fixtures then I can rollback my changes here.

Comment (atwixfirster) [reply dev/tests/integration/testsuite/Magento/Sales/_files/guest_quote_with_addresses.php]: > We could not change current fixtures

**fixed**

Thank you, @naydav !!

**Pull request merged**